loan/src/test/java/com/iluwatar/loan/MainTest.java

```diff
@@ -0,0 +1,47 @@
+package com.iluwatar.loan;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest {
```

Comment:
We usually have AppTest that runs the example and checks there are no errors. Then we have a set of unit tests.